packages/plugin-doc-coverage/mocks/component-mock.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
Maybe lets either remove this mocks if they are not used or move them into a folder like the angular example

Reply:
Mmm yeah, barely used, and in the Angular fixture there is already every case